sematic/ui/packages/common/src/pages/RunDetails/PipelineSection.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from '@emotion/styled';
+import Typograph from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import MoreVertButton from 'src/component/MoreVertButton';
+
+import theme from 'src/theme/new';
+
+const HeadLine = styled.h2`
+    margin: 0;
+`;
+
+const TopSection = styled.section`
+    height: 100px;
+    position: relative;
+`;
+
+const BottomSection = styled.section`
+    height: 100px;
+    border-bottom: 1px solid ${theme.palette.p3border.main};
+    display: flex;
+    margin: 0 -25px;
+    & .MuiBox-root {
+        width: 100px;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+    }
+`;
+
+const StyledVertButton = styled(MoreVertButton)`
+    position: absolute;
+    right: 0;
+    top: 12px;
+    transform: translate(50%,0);
+`;
+
+const PipelineSection = () => {
+    return <>
+        <TopSection>
+            <HeadLine>PIPELINE</HeadLine>
+            <Typograph variant='bold' style={{ marginTop: 15, marginBottom: 12 }}>MNIST PyTorch Example</Typograph>
```

Comment:
We need to come up with strategies for when this text and the import path below are too long. I see three options:

For the name "A very very long pipeline name with many words", and import path "the.longest.import.path.you.have.ever.seen":


1. Trim the beginning: "... name with many words" and "...you.have.ever.seen"
2. Trim the end: "A very very long pipeline ..." and "the.longest.import.path..."
3. Trim the middle: "A very very long ... with many words" and "the.longest....have.ever.seen"

The easiest and most straightforward is likely option 2. When the label is trimmed, we can add a tooltip on the ellipsis (or on the full label) that will display the full label on hover.

Any thoughts?

Reply:
As discussed, I went with option 2. Also added a tooltip if the ellipsis is displayed. In case the ellipsis is not displayed (depending on the length of the text), the tooltip will be disabled.